apps/voting/contracts/Voting.sol

```diff
@@ -128,6 +132,20 @@ contract Voting is IForwarder, AragonApp {
         emit ChangeMinQuorum(_minAcceptQuorumPct);
     }
 
+    /**
+    * @notice `_earlyExecutionAllowed? 'Allow' : 'Disallow'` votes early execution
+    * @param _earlyExecutionAllowed New votes early execution setting
+    */
+    function changeEarlyExecution(bool _earlyExecutionAllowed)
+        external
+        authP(MODIFY_EARLY_EXECUTION_ROLE, arr(uint256(_earlyExecutionAllowed ? 1 : 0), uint256(earlyExecutionDisallowed ? 0 : 1)))
+    {
+        bool _earlyExecutionDisallowed = !_earlyExecutionAllowed;
```

Comment:
Could we rename this to `earlyExecutionDisallowed_` instead, to avoid confusion with arguments and local variables?